### DISCUSSION TITLE
Long audio file crashing the browser tab

### DISCUSSION THREAD

**Discussion by Jagat-Brilworks** [katspaugh/wavesurfer.js#3648] [Q&A]
I am trying to load a 120-minute audio file in the below code but my browser tab always crashed.

```javascript
const duration = getSpeedDuration(track.duration, track.speed);
const audioObject = WaveSurfer.create({
  container: `#wave-1`,
  url: track?.url,
  autoScroll: false,
  backend: 'MediaElement',
  autoCenter: true,
  cursorColor: 'transparent',
  waveColor: '#A1A5DF',
  progressColor: '#9244eb',
  width: Math.ceil(duration * 20),
  minPxPerSec: getSpeedDuration(20, track.speed),
  interact: false,
  normalize: true,
});
``` 
![image](https://github.com/katspaugh/wavesurfer.js/assets/118892325/4fa9a27e-b805-4bbf-b9a4-f8b4db97f3e9)

**Comment by nirmalrathodbril** [comment]
I do have same issue

**Comment by RBShaktiSingh** [comment]
I do have same issue multiple times.

**Comment by katspaugh** [comment]
It's covered in the FAQ in the readme: https://github.com/katspaugh/wavesurfer.js?tab=readme-ov-file#faq

Apart from that, please don't create question issues. The issue template specifically asks to not create question issues.
